src/wasm/wasm-binary.cpp

```diff
@@ -1953,6 +2096,10 @@ void WasmBinaryReader::readCustomSection(size_t payloadLen) {
     readDylink(payloadLen);
   } else if (sectionName.equals(BinaryConsts::CustomSections::Dylink0)) {
     readDylink0(payloadLen);
+  } else if (sectionName.equals(Annotations::BranchHint.str)) {
```

Comment:
Since both sides are `Name`, can't we use `==`?
```suggestion
  } else if (sectionName == Annotations::BranchHint) {
```

Reply:
Done.